### PR TITLE
Updated implementation based on change in CogComp's Pipeline

### PR DESCRIPTION
**Pull request by GHLgh**

### CODE REVIEW

File: tests/test_pipeline_config.py [hunk 1]
```diff
@@ -20,103 +20,46 @@ def setUp(self):
...
-SHALLOW_PARSE = false
-SRL_VERB = false
-DEPENDENCY_STANFORD = false
-DEPENDENCY = false
-PARSE_STANFORD = false
-SRL_PREP = false
-
 [remote_pipeline_setting]
-api = http://austen.cs.illinois.edu:8080/annotate
+api = http://austen.cs.illinois.edu:7080
```
Comment (danyaljj) [inline tests/test_pipeline_config.py]: this is "5800"?

File: tests/test_pipeline_config.py [hunk 2]
```diff
@@ -20,103 +20,46 @@ def setUp(self):
...
+        self.assertEqual(config['remote_pipeline_setting']['api'], 'http://austen.cs.illinois.edu:8080')
 
     @mock.patch('sioux.pipeline_config.download')
     def test_get_current_config_with_models(self, mock_dl):
         mock_dl.get_root_directory.return_value = test_config_folder
         mock_dl.get_model_path.return_value = test_config_folder
         config, models = sioux.pipeline_config.get_current_config()
         self.assertEqual(True, models)
-        self.assertEqual(config['local_pipeline_setting']['LEMMA'], 'true')
+        self.assertEqual(config['remote_pipeline_setting']['api'], 'http://austen.cs.illinois.edu:7080')
```
Comment (danyaljj) [inline tests/test_pipeline_config.py]: "5800"?

File: tests/test_pipeline_config.py [hunk 3]
```diff
@@ -20,103 +20,46 @@ def setUp(self):
...
-        self.assertEqual(config['local_pipeline_setting']['LEMMA'], 'true')
 
     @mock.patch('sioux.pipeline_config.download')
     def test_get_current_config_without_models(self, mock_dl):
-        mock_dl.get_root_directory.return_value = test_config_folder
         mock_dl.get_model_path.return_value = 'not_exists_folder_path'
         config, models = sioux.pipeline_config.get_current_config()
         self.assertEqual(False, models)
-        self.assertEqual(config['local_pipeline_setting']['LEMMA'], 'false')
+        self.assertEqual(config['remote_pipeline_setting']['api'], 'http://austen.cs.illinois.edu:8080')
```
Comment (danyaljj) [inline tests/test_pipeline_config.py]: "5800"

File: tests/test_pipeline_config.py [hunk 4]
```diff
@@ -20,103 +20,46 @@ def setUp(self):
...
 
     @mock.patch('sioux.pipeline_config.download')
     def test_get_user_config(self, mock_dl):
         # such that will use package config if user provided file does not exist
-        mock_dl.get_root_directory.return_value = test_config_folder
         mock_dl.get_model_path.return_value = 'not_exists_folder_path'
         config, models = sioux.pipeline_config.get_user_config(test_config_folder+'/config.cfg')
-        self.assertEqual(False, models)
-        self.assertEqual(config['local_pipeline_setting']['LEMMA'], 'true')
+        self.assertEqual(config['remote_pipeline_setting']['api'], 'http://austen.cs.illinois.edu:7080')
```
Comment (danyaljj) [inline tests/test_pipeline_config.py]: "5800"?

File: tests/test_pipeline_config.py (hunk 1 above)
Comment (GHLgh) [inline tests/test_pipeline_config.py]: It doesn't matter, tests for pipeline_config only check if the module loads config correctly

**Pull request merged**